### PR TITLE
Add docstring details in epoch readers

### PR DESCRIPTION
**Pull request by nabilalibou**
Small fix for #12542 (DOC, no-changelog-entry-needed)

### CODE REVIEW

File: mne/io/eeglab/eeglab.py [hunk 1]
```diff
@@ -385,8 +385,8 @@ def read_epochs_eeglab(
 
     Returns
     -------
-    epochs : instance of Epochs
-        The epochs.
+    EpochsEEGLAB : instance of BaseEpochs
+        The epochs EpochsEEGLAB (subclass of BaseEpochs).
```
Comment (hoechenberger) [inline mne/io/eeglab/eeglab.py:389]: i don't think this technical description is helpful here, i would revert this line

File: mne/io/kit/kit.py [hunk 2]
```diff
@@ -1025,8 +1025,12 @@ def read_epochs_kit(
 
     Returns
     -------
-    epochs : instance of Epochs
-        The epochs.
+    EpochsKIT : instance of BaseEpochs
+        The epochs EpochsKIT (subclass of BaseEpochs).
```
Comment (hoechenberger) [inline mne/io/kit/kit.py:1029]: same

File: mne/io/eeglab/eeglab.py (hunk 1 above)
Comment (nabilalibou) [reply mne/io/eeglab/eeglab.py]: Thx for the feedback, the redundant description has been removed

**Comment by hoechenberger** [comment]
Thanks @nabilalibou!

I'll leave it to @drammock or @larsoner to merge

**Review by drammock — APPROVED** [review]

**Pull request merged**